molecule_libvirt/playbooks/templates/network/network.xml.j2

```diff
@@ -0,0 +1,10 @@
+<network>
+  <name>{{ molecule_network_name }}</name>
+  <bridge name="molecule0"/>
+  <forward/>
+  <ip address="10.10.10.1" netmask="255.255.255.0">
```

Comment:
The range should be a variable to avoid possible conflict.

Reply:
The user will have to add `molecule_network_cidr: "10.10.10.0/24"` now. All other values will be deducted from this variable.